docs/contributor-guide/developer-guide/deploy-system/prerequisits.md

```diff
@@ -0,0 +1,65 @@
+# Prerequisits and developer setup
+
+This page contains instructions for the setup you should go through before you start development work on the deploy system.
+It is intended as a step by step guide and you are encouraged to improve the documentation if any step is not straight forward in its current state.
+
+ TODO([#4324](https://github.com/civiform/civiform/issues/4324))This entire page needs to be rewritten. For each item in the list explain what it is and how to install/get it, remove or change the existing documentation below as required. 
+ * Access you need to ask for to not get blocked on later steps
+ * Required tools(python3, terraform etc, pip, yapf)
+ * AWS accounts(including differences depending on who your employer is)
+ * AWS CLI
+ * AWS nuke
+ * Git setup for cross repository development 
+    * for developing and running directly from deploy-infra 
+    * for running from deploy like an civic entity sre would
+
+### Before running scripts
+1. Required to be installed: bash, python, cloud provider CLIs.
+2. User needs to be authenticated with cloud provider.
+    * AWS: run `aws configure` or export `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
+    * Azure: run `az login`
+
+## Development setup for developent across git branches
+
+Most deployment system changes happen in the cloud-deploy-infra repository. The deployment script however is located in the civiform-deploy
+Because of the code being located in different git repositories
+The setup.py script starts the deployment in the civiform-deploy repository, but most of the code is in the cloud-deploy-infra repository. Therefore additional steps have to be taken so changes made to the cloud-deploy-infra repository are visible when you run the setup.py script: 
+
+### Option 1(current, to be deprecated): Direct checkout
+Use git to copy a part of the cloud-deploy-infra repository into the civiform-deploy repository before running the setup.sh script:
+
+All scripts currently do the following:
```

Comment:
Is this section meant to be a high level overview of what's happening or instructions for the developer on what to do? I think it's the former but it's a little confusing at the moment, so might be worth stating the context explicitly.

Reply:
This entire rest of the page is the documentation that was in place before I started updating. I left it in here until we have a replacement for it. I have clarified the TODOs to make this clear. I'd expect most it to be either extended significantly or removed.